src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map((tab) => {
+          const { id, title } = tab;
+
+          return (
+            <li
+              className={cn(tab === selectedTab && 'is-active')}
+              data-cy="Tab"
+              key={id}
+            >
+              <a
+                href={`#${id}`}
+                data-cy="TabLink"
+                onClick={() => {
+                  if (tab !== selectedTab) {
+                    onTabSelected(tab);
+                  }
+                }}
```

Comment:
create a separate handler for this function for better readability and maintainability